src/main/java/edu/rosehulman/quota/controller/GetQuotaSDKController.java

```diff
@@ -11,7 +11,7 @@
 
 import static spark.Spark.halt;
 
-public class GetQuotaController implements Route {
+public class GetQuotaSDKController implements Route {
 
   @Override
   public Object handle(Request request, Response response) throws Exception {
```

Comment:
So this is fine, you use `apiKey` here, but in `Application.java` you use `partnerId`, so you can change it in `Application`